script.js

```diff
@@ -268,5 +268,10 @@ $(document).ready(function() {
     
       return 0;
     }
+//page animations
+    $(".panel").slideDown(1500);
+    $(".columns").fadeIn(7000);
```

Comment:
On page load, the columns will take 7 seconds to fade in. If the user enters an artist immediately, they have to wait about 6ish seconds for the page to fade in, which is probably too long. If the user waits on the landing page for more than 7 seconds, the animation will not be shown to the user because it already happened in the background (it was just hidden).